src/current/v22.2/cockroach-workload.md

```diff
@@ -97,7 +97,7 @@ Flag | Description
 `--ramp` | The duration over which to ramp up load.<br><br>**Applicable command:** `run`
 `--ranges` | The initial number of ranges in the `bank` table.<br><br>**Applicable commands:** `init` or `run`<br>**Default:** `10`
 `--rows` | The initial number of accounts in the `bank` table.<br><br>**Applicable commands:** `init` or `run`<br>**Default:** `1000`
-`--seed` | The key hash seed.<br><br>**Applicable commands:** `init` or `run`<br>**Default:** `1`
+`--seed` | The key hash seed.<br><br>**Applicable commands:** `init` or `run`<br>**Default:** A random seed is used each time the command is run. Reusing a seed is recommended only for reproducing a specific pattern or behavior, but workloads that have a random element should use the default random seed.
```

Comment:
```suggestion
`--seed` | The key hash seed.<br><br>**Applicable commands:** `init` or `run`<br>**Default:** A random seed is used each time the command is run. Reusing a seed is recommended only for reproducing a specific pattern or behavior. Workloads that have a random element should use the default random seed.
```

I would separate these into two sentences as it doesn't feel like they need to have the comparison with "but". This is how I read it — but I could understand an argument against this.

Reply:
I took your suggestion. I guess if it makes it look like we are recommending using non-random seeds, we will get feedback about it.